department-of-veterans-affairs/caseflow
# Make cancelled appeal obvious in case search

Resolves #13654

### Description
Pass ama appeal status to the front end to be shown in case search. Do not show appeal assignee if it is cancelled. Color status red for cancelled appeals.

### Acceptance Criteria
- [x] No assignee is shown for cancelled appeals in case search
- [x] Cancelled appeals show a red status of cancelled appeals in case search

### Testing Plan
1. Navigate to http://localhost:3000/search?veteran_ids=377
1. Cancel the appeal
```ruby
Appeal.find_by(uuid: "a7654f2e-3aa1-4efa-b671-ce642d43e455").tasks.each(&:cancelled!)
```
1. Ensure these is no assignee listed
1. Ensure the case status is "cancelled" and red

### User Facing Changes

Before|After
---|---
![Screen Shot 2020-04-09 at 12 46 58 PM](https://user-images.githubusercontent.com/45575454/78919832-7e24a000-7a60-11ea-8257-fbab02ca2fa9.png)|![Screen Shot 2020-04-09 at 12 45 53 PM](https://user-images.githubusercontent.com/45575454/78919845-811f9080-7a60-11ea-9b4b-9d1eaffa85c4.png)
